ddln_imaginator.proto

```diff
@@ -183,6 +183,23 @@ message WorldInfo {
     optional LatLongAlt home_geo = 4;
 }
 
+// World intersection request with a 3D ray.
+message WorldIntersectionRequest {
+    repeated Ray3d rays_W = 1;
+}
+
+// World intersection response: first point encountered in the world.
```

Comment:
Consider clarifying explicitly what parts of the world participate in intersections and what are not.

IIUC, only terrain participates but sky + clouds + objects do not?

Reply:
You are right, but I intentionally copied these sections exactly as they are in https://git.daedalean.ai/exp.git/tree/imaginator/imaginator.proto

I think it's better to avoid inconsistency between these files, even in comments. I've added a relevant node to the `SitlService` instead.